aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1017,14 +1016,19 @@ static void addmm_impl_cpu_(
   if (m1_strides[transpose_c ? 1 : 0] == 1 &&
       m1_strides[transpose_c ? 0 : 1] >= std::max(int64_t{1}, m)) {
     transpose_a = false;
-    a = m1;
+    a = m1.resolve_conj();
   } else if (m1_strides[transpose_c ? 0 : 1] == 1 &&
              m1_strides[transpose_c ? 1 : 0] >= std::max(int64_t{1}, k)) {
     transpose_a = true;
     a = m1;
   } else {
     transpose_a = !transpose_c;
-    a = m1.clone(at::MemoryFormat::Contiguous);
+    if (m1.is_conj() && transpose_a) {
+      a = m1.conj().clone(at::MemoryFormat::Contiguous);
```

Comment:
why do you not expect any performance gain by using `ConjTranspose` here? is it because the conjugation is resolved in clone and so there's no additional kernel call?

Reply:
Yes. You have to do `.clone` either way, and then, whether you send `Transpose` or `ConjTranspose` to blas, blas performance itself doesn't change. So this special casing doesn't make things worse, but it also doesn't make them better and adds code complexity.